system-tests/universe/test-universe.js

```diff
@@ -243,7 +243,18 @@ describe("Universe", function() {
     });
 
     it("plan is displayed", () => {
-      cy.visitUrl(`services/detail/%2F${packageName}/plans`);
+      cy.visitUrl(`services/detail/%2F${packageName}/tasks`);
```

Comment:
system test didnt account for schedule still deploying and plans request timing out. this should fix it. (EE CI is red b/c of that)

Reply:
@juliangieseke :+1:, but your commit message was too long and fail the commit linter. I'm amending it and force pushing to get a ✅ build :)

Reply:
narf… why didn't my local tooling catch it? 🤔 /cc @DanielMSchmidt 

nvm, thx 4 fixing 👍